backoff.py

```diff
@@ -218,10 +218,22 @@ def constant(interval=1):
         yield interval
 
 
+def random_jitter(value):
+    return value+random.random()
```

Comment:
This also passed in my Macbook, and online checker: http://pep8online.com/s/zxKQyxsK

Would you mind to point out the failure?